explanation/CalibratedFPSPhot.ipynb

```diff
@@ -1081,9 +1081,14 @@
+    "By default, the baseline for each `fcqfid` is determined using observations obtained more than *100 d prior* to the time of maximum light and after when transients have faded to 22.5 mag calculated based on $^{56}$Co decay rate. If there are no observations near the time of maximum light for the calculation, observations obtained 611 days after the time of maximum light are used. These thresholds are extremely conservative, though there are transients that have long rise times or that can be detected for years after they are discovered. Warnings are implemented in order to capture such instances. The baseline correction `C` is calculated via the median `forcediffimflux` value in the baseline region, and uncertainty of the baseline `C` is obtained using bootstrapping.\n",
     "\n",
-    "We report the total number of observations used to calculate `C` and whether the baseline is measured in observations before or after maximum light. We advise caution when using light curves with few baseline observations (< ~25), and do not provide baseline corrections when there are < 10 observations in the baseline region. If there are > 25 pre-maximum baseline observations or (> 10 pre-maximum AND < 25 post-maximum baseline observations then we use the pre-maximum baseline. Otherwise we use the post-maximum baseline. Pre-maximum baseline measurements are always better than post-maximum measurements because there is always some transient emission present at late times (emission that is statistically insignificant after 500 d, hopefully). "
+    "We report the total number of observations used to calculate `C` and whether the baseline is measured in observations before, after, or before and after the maximum light. We advise caution when using light curves with few baseline observations (< ~25), and do not provide baseline corrections when there are < 2 observations in the baseline region. If there are < 10 observations in the pre-maximum light baseline observations and < 10 observations in the post-maximum light baseline observations, we apply the following criteria:\n",
+    "- if there is no pre-maximum light baseline observations, post-maximum light baseline observations are used to measure the baseline\n",
+    "- if there are < 5 pre-maximum light baseline observations and more than 10 post-maximum light baseline observations, post-maximum light baseline observations are used to measure the baseline\n",
+    "- if the maximum snr of the pre-maximum light baseline observations is > 6, a \"long rise\" warning is given\n",
```

Comment:
we need to adjust this, I think we can be more generic and say - if there emission in the baseline then we provide the ... warning